Pipelines/Redirects.cs

```diff
@@ -42,6 +42,8 @@ public Redirects()
                 { "production/community", "https://chat.whatsapp.com/J3QApbmJlMZ9oYrlVGGkGU" },
                 { "justso/material", "https://drive.google.com/drive/folders/12jf3KdeXp0iSfh9rsE5Nd2Lo62lzzCNX" },
                 { "justso/form", "https://docs.google.com/forms/d/e/1FAIpQLSdAtaSZ4AoiN_x_ilttVSq-wJthyeCCHxBaS9IxrpFm9GGhZQ/viewform" },
+                { "september-2024-pitch-form", "https://docs.google.com/document/d/1tCxS9i3uRrWv7ksV6L_Yn_aUC5a0B40K9R9Da3c_AOY/edit" },
```

Comment:
These links don't appear to be accessible anonymously, and I'm not sure the edit link is right...

Reply:
@olevett That is (currently) intended - I want to get the placeholder links in before sending to committee - the docs are currently restricted. The `/edit` form is what we previously have (and will work globally when the permissions become view). At the moment, whilst under discussion, they're restricted.

Reply:
👍  🚢